docs/topics/forms/index.txt

```diff
@@ -559,13 +559,73 @@ the :meth:`.Form.render`. Here's an example of this being used in a view::
 
 See :ref:`ref-forms-api-outputting-html` for more details.
 
+.. _reusable-field--group-templates:
+
+Reusable field group templates
+------------------------------
+
+.. versionadded:: 5.0
+
+Each field is available as an attribute of the form, using ``{{
+form.name_of_field }}`` in a template. A field has a
+:meth:`~django.forms.BoundField.as_field_group` method which renders the
+related elements of the field as a group, its label, widget, errors and help
+text.
+
+This allows generic templates to be written that arrange fields elements in the
+required layout. For example::
+
+.. code-block:: html+django
+
+    {{ form.non_field_errors }}
+    <div class="fieldWrapper">
+    {{ form.subject.as_field_group }}
+    </div>
+    <div class="fieldWrapper">
+    {{ form.message.as_field_group }}
+    </div>
+    <div class="fieldWrapper">
+    {{ form.sender.as_field_group }}
+    </div>
+    <div class="fieldWrapper">
```

Comment:
Can we use `field-wrapper` to avoid adding code words to the `spelling_wordlist`?